apps/condo/domains/miniapp/schema/B2BAppPermission.test.js

```diff
@@ -188,9 +188,9 @@ describe('B2BAppPermission', () => {
                     expect(allPermissions).toHaveLength(0)
                 })
             })
-            test('No one in the organization can, including the administrator', async () => {
-                const allPermissions = await B2BAppPermission.getAll(manager, { id: permission.id })
-                expect(allPermissions).toHaveLength(0)
+            test('Organization employee can', async () => {
```

Comment:
Are you really need to give access to all employees?
Seems like ones with `canManageRoles` should be able to see this